kartei.cls

```diff
@@ -105,14 +105,14 @@
 \DeclareOption{a7paper}{%
 	\card@papersize=2%
 	\PassOptionsToPackage{paper=a7,pagesize}{typearea}%
-	\setlength{\card@hoffset}{-27pt}\setlength{\card@voffset}{42.5pt}%
+	\setlength{\card@hoffset}{-33.75pt}\setlength{\card@voffset}{37pt}%
 }%
 %A8 Cards
 \DeclareOption{a8paper}{
 	\card@papersize=3%
 	\PassOptionsToPackage{paper=a8,pagesize}{typearea}%
 	\setlength{\card@hoffset}{-33.75pt}\setlength{\card@voffset}{37pt}%
-}%
```

Comment:
Bitte keine Leerzeichen am Zeilenende.

Reply:
Gibt es auch eine Möglichkeit sowas in GitHub anzupassen?